### ISSUE TITLE
Offset values in glmer (family=binomial) that were not specified by the user 

### ISSUE THREAD

**Issue by heliosdrm** [lme4/lme4#334]
I have come across a seemingly inconsistent behavior of glmer between the 64-bit and 32-bit version of R. I'm using the data frame "dd" uploaded here:
https://gist.github.com/heliosdrm/41742604ebd3dc2ca36f

The code I used is here:

```R
> library(lme4)
Loading required package: Matrix
Warning messages:
1: package 'lme4' was built under R version 3.2.2
2: package 'Matrix' was built under R version 3.2.2
> m <- glmer(y~f+(f|subj),data=dd,family=binomial,
+ control=glmerControl(optimizer="bobyqa"))
```

The problem is: when I use R 64-bit, the model contains offset values that I did not expect - as obtained by `getME(m, "offset")` . The offset is zero if I use R 32-bit (in the same machine, same packages), or with the following small variations in the model fit:

- Leave only the random effect on the intercept, i.e. the random term of the formula equal to `(1|subj)`.
- Leave the argument `control` unspecified.

A curious thing is that the offsets are equal to the values computed with the fixed coefficients:

```R
> # Values given by the coefficients of the fixed effects
> cumsum(fixef(m))
(Intercept)          fb
  -3.254889   -2.917308
> # Offset for each level
> aggregate(getME(m,"offset"),dd["f"],mean)
  f         x
1 a -3.254889
2 b -2.917308
```

Is this a bug of the 64-bit version? Or what explanation has this?

```R
> sessionInfo()
R version 3.2.0 (2015-04-16)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 7 x64 (build 7601) Service Pack 1

locale:
[1] LC_COLLATE=Spanish_Spain.1252  LC_CTYPE=Spanish_Spain.1252
[3] LC_MONETARY=Spanish_Spain.1252 LC_NUMERIC=C
[5] LC_TIME=Spanish_Spain.1252

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base

other attached packages:
[1] lme4_1.1-9   Matrix_1.2-2

loaded via a namespace (and not attached):
[1] minqa_1.2.4     MASS_7.3-40     tools_3.2.0     Rcpp_0.12.0
[5] splines_3.2.0   nlme_3.1-120    grid_3.2.0      nloptr_1.0.4
[9] lattice_0.20-31
>
```
Thanks.

**Comment by bbolker** [comment]
do we have *any* guesses at all why this would manifest itself differently on 32- vs 64-bit OSes?

**Comment by bbolker** [comment]
I can't easily reproduce this, but I think it is fixed by  3c0fa76 ... can someone who *can* reproduce test this ... ?
